package.json

```diff
@@ -187,6 +187,7 @@
     "jest": "^28.1.1",
     "jest-environment-jsdom": "28.1.1",
     "jquery": "1.11.3",
+    "jung": "^2.1.0",
```

Comment:
Where is this being used?

Reply:
its used in a few of the package.json scripts.  I use it for `npm run cdn:watch` and it started failing because the dependency got removed, just added it back